sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -222,6 +225,40 @@ TEST_F(Key256Test, start) {
   EXPECT_EQ(dif_aes_start(&aes_, &transaction, kKey_, NULL), kDifOk);
 }
 
+// Manual operation
+class ManualOperationTest : public AesTestInitialized {
+ protected:
+  ManualOperationTest() {
+    transaction.manual_operation = kDifAesManualOperationManual;
+  }
+};
+
+TEST_F(ManualOperationTest, start) {
+  EXPECT_READ32(AES_STATUS_REG_OFFSET, 1);
+  setExpectedConfig(AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_128,
+                    AES_CTRL_SHADOWED_MODE_VALUE_AES_ECB,
+                    AES_CTRL_SHADOWED_OPERATION_VALUE_AES_ENC, true);
```

Comment:
I am not entirely following how this test works but since it got mixed up in the past: does this test indeed test the MANUAL_OPERATION bit or just the OPERATION bit? These are two different fields in the register and they have a completely different meaning.

Reply:
I changed it to be clear to read. Let me know if it is still confusing you.

Reply:
Thanks, @engdoreis it is more clear to me now.